From ee12f665f33467b359bdec69110cc33f84850643 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 31 Dec 2024 21:31:10 +0800
Subject: [PATCH 173/286] AOSCOS: hp-wmi: add 8BB3 to
 omen_thermal_profile_boards

Mark OMEN by HP Transcend Gaming Laptop 16-u0xxx (board ID: 8BB3) for OMEN
thermal profiles to fix low power draw issue.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Link: https://t.me/c/1109254909/695630
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/x86/hp/hp-wmi.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/platform/x86/hp/hp-wmi.c b/drivers/platform/x86/hp/hp-wmi.c
index 95e65073b4b3..f43391b63d4b 100644
--- a/drivers/platform/x86/hp/hp-wmi.c
+++ b/drivers/platform/x86/hp/hp-wmi.c
@@ -64,7 +64,8 @@ static const char * const omen_thermal_profile_boards[] = {
 	"874A", "8603", "8604", "8748", "886B", "886C", "878A", "878B", "878C",
 	"88C8", "88CB", "8786", "8787", "8788", "88D1", "88D2", "88F4", "88FD",
 	"88F5", "88F6", "88F7", "88FE", "88FF", "8900", "8901", "8902", "8912",
-	"8917", "8918", "8949", "894A", "89EB", "8BAB", "8BAD", "8A42", "8A15"
+	"8917", "8918", "8949", "894A", "89EB", "8BAB", "8BAD", "8A42", "8A15",
+	"8BB3"
 };
 
 /* DMI Board names of Omen laptops that are specifically set to be thermal
-- 
2.48.0.rc1

